docs/getting-started/installation/binaries.md

```diff
@@ -173,19 +194,42 @@ Download and execute the install script:
 > Notice: version is recommended to use the latest version.
 
 ```bash
-curl \
-  --proto '=https' \
-  --tlsv1.2 -L -o client-{arch}-unknown-linux-gnu.deb \
-  https://github.com/dragonflyoss/client/releases/download/v{version}/client-v{version}-{arch}-unknown-linux-gnu.deb
+wget -O  dragonfly-client-{arch}-unknown-linux-musl.deb
+https://github.com/dragonflyoss/client/releases/download/v{version}/dragonfly-client-v{version}-{arch}-unknown-linux-musl.deb
+
 
-dpkg -i client-{arch}-unknown-linux-gnu.deb
+dpkg -i dragonfly-client-{arch}-unknown-linux-musl.deb
 ```
 
 Make sure to replace `arch` with one of the following:
 
 - `x86_64`
 - `aarch64`
 
+Modify your `dfdaemon.yaml` (default location: /etc/dragonfly/dfdaemon.yaml), please refer to [Dfdaemon](../../reference/configuration/client/dfdaemon.md).
```

Comment:
Modify your `dfdaemon.yaml`, the default path is `/etc/dragonfly/dfdaemon.yaml` in linux, please refer to [dfdaemon config](../../reference/configuration/client/dfdaemon.md).